packages/expo-notifications/src/useLastNotificationResponse.ts

```diff
@@ -5,11 +5,39 @@ import { addNotificationResponseReceivedListener } from './NotificationsEmitter'
 import NotificationsEmitterModule from './NotificationsEmitterModule';
 
 /**
- * Return value of this hook may be one of three types:
- * - `undefined` until we know what to return
- * - `null` until the app receives any notification response
- * - an object of `NotificationResponse` type - the response
- *   that has been received by the app most recently.
+ * A React hook always returns the notification response that was received most recently
+ * (a notification response designates an interaction with a notification, such as tapping on it).
+ *
+ * > If you don't want to use a hook, you can use `Notifications.getLastNotificationResponseAsync()` instead.
+ *
+ * @return The hook may return one of these three types/values:
+ * - `undefined` -- until we're sure of what to return,
+ * - `null` -- if no notification response has been received yet,
+ * - a [`NotificationResponse`](#notificationresponse) object -- if a notification response was received.
```

Comment:
```suggestion
 * - `undefined` - until we're sure of what to return,
 * - `null` - if no notification response has been received yet,
 * - a [`NotificationResponse`](#notificationresponse) object - if a notification response was received.
```

Let's use a single hyphen or an em dash here.